taichi/backends/opengl/opengl_api.cpp

```diff
@@ -236,7 +244,8 @@ GLSSBO *root_ssbo;
 void create_glsl_root_buffer(size_t size) {
   // if (root_ssbo) return;
   initialize_opengl();
-  root_ssbo = new GLSSBO;  // TODO(archibate): mem leaking
+  root_ssbo = new GLSSBO;  // TODO(archibate): mem leaking, use std::optional instead
```

Comment:
A very little change (< 10 CoL) can fix this mem leakage, should I put in this PR?

Reply:
If it's just 10 LoC then let's fix it here.

Reply:
Ok.